src/containers/navigation/components/Wallet/WalletBalanceMarkup.tsx

```diff
@@ -15,7 +15,7 @@ import { useTariBalance } from '@app/hooks/wallet/useTariBalance.ts';
 import { toggleHideWalletBalance } from '@app/store/actions/uiStoreActions.ts';
 import { useUIStore } from '@app/store';
 
-export default function WalletBalanceMarkup() {
+const WalletBalanceMarkup = memo(function WalletBalanceMarkup() {
```

Comment:
If this value is memoized. Won't it never update again?

Reply:
its children change so that counts as a prop update, and it was updating while i tested, but lemme double check :D

Reply:
https://github.com/user-attachments/assets/e9f49635-ce56-4048-86c7-f17d0323a7c2

yep updating normally :D

Reply:
Probably because JS doesn't know what memoize means